--- arch/arm/board/generic/dts/arm/vexpress-v2p-ca9.dts	2019-05-19 21:22:24.426706326 +0300
+++ ./vexpress-v2p-ca9.dts	2019-05-19 21:24:06.089786093 +0300
@@ -16,11 +16,11 @@
 		bootcmd = /* Mount initrd device */
 			  "vfs mount initrd /",
 
-			  /* Run boot script */
-			  "vfs run /boot.xscript",
-
 			  /* Print banner */
-			  "vfs cat /system/banner.txt";
+			  "vfs cat /system/banner.txt",
+
+			  /* Run boot script */
+			  "vfs run /boot.xscript";
 	};
 
 	aliases { };
